db/data_migrations/20121204143748_add_hidden_commodities.rb

```diff
@@ -19,6 +19,7 @@
   up do
     applicable {
       HiddenGoodsNomenclature.where(goods_nomenclature_item_id: CODES).count != CODES.size
```

Comment:
We might still need hidden gonos I will need to check

Reply:
I'll import them together with chapters & notes vis sql dump

Reply:
+ measurement_unit_abbreviations